application/frontend/controllers/AuthController.php

```diff
@@ -86,35 +117,40 @@ public function actionLogin()
 
     public function actionLogout()
     {
-        if (\Yii::$app->user->isGuest) {
+        $accessToken = \Yii::$app->request->get('access_token');
+        if ($accessToken !== null) {
```

Comment:
We don't use sessions for identity/authorization anymore, just the token, so the logout method needs the token to nullify the token in the db and redirect to IdP to kill session there too.